css/style.css

```diff
@@ -1,12 +1,112 @@
-#navbar {
-    position: fixed;
-    top: 0;
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700&display=swap");
+
+* {
+  font-family: "Open Sans", sans-serif;
+}
+
+html,
+body {
+  margin: 0;
+  width: 100%;
+  height: 100%;
+}
+
+body {
+  background: #0799d5;
+}
+
+#formpage {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+form {
+  background-color: #3a469c;
+  border-radius: 10px;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+#title,
+#description,
+label {
+  color: white;
+}
+
+#title {
+  margin-top: 30px;
+  margin-bottom: 10px;
+}
+
+#description {
+  margin-top: 0;
+  margin-bottom: 30px;
+}
+
+#name,
+#email,
+#number,
+#dropdown {
+  min-width: 400px;
+}
+
+#name,
+#email,
+#number,
+#dropdown,
+#textarea {
+  border: 3px solid #0799d5;
+}
+
+#name,
+#email,
+#number {
+  height: 25px;
+}
+
+#dropdown {
+  padding-top: 2px;
+  padding-bottom: 2px;
+  width: 100%;
+}
+
+label {
+  margin-bottom: 10px;
+}
+
+textarea {
+  resize: none;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.turn-info-fc {
+  padding: 5px;
+  margin: 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
 }
-#welcome-section {
-    height: 100vh;
+
+#button-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
 }
 
+button {
+  border: 1px solid #f8f9fa;
+  font-size: 14px;
+  height: 36px;
+  border-radius: 4px;
+  color: #3c4041;
+  background-color: #f8f9fa;
+  margin: 25px;
+}
 
-@media (min-width: 400px) {
```

Comment:
please do not leave unused code in your codebase because can make your code not clean, please can remove this line, it will be easier to understand your code.